From 3dd76b6742b1e5c3576a609096e6d57740c422df Mon Sep 17 00:00:00 2001
From: Li Jun <B47624@freescale.com>
Date: Sat, 27 Sep 2014 21:07:48 +0800
Subject: [PATCH] MLK-9638-2 usb: otg: test: add otg_hnp_reqd flag in struct
 otg_fsm

This patch adds flag otg_hnp_reqd in struct otg_fsm to indicate A device
as host wants to start HNP sequence.

Acked-by: Peter Chen <peter.chen@freescale.com>
Signed-off-by: Li Jun <b47624@freescale.com>
(cherry picked from commit 92382e06c6902731b05dcbb0db4f0ebfa486969f)
---
 include/linux/usb/otg-fsm.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/usb/otg-fsm.h b/include/linux/usb/otg-fsm.h
index 2f02d14..3621c77 100644
--- a/include/linux/usb/otg-fsm.h
+++ b/include/linux/usb/otg-fsm.h
@@ -113,6 +113,7 @@ struct otg_fsm {
 	int tst_maint;
 	int otg_vbus_off;
 	int otg_srp_reqd;
+	int otg_hnp_reqd;
 
 	/* Informative variables */
 	int a_bus_drop_inf;
-- 
1.8.0

